Pull Request for Universal Phone Mount - Voron 2.4r2

AVieira-1 opened the pull request:
<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!
-->

<!--
Describe the submission further using the template provided below. The more
details the better! Please don't forget to thoroughly read the mod requirements and rules
and make sure your mod submission adheres to these rules.
Doing so speeds up the review process and will increase your chances at acceptance.
-->

### Which mods are added by this PR?
Universal_Phone_Mount
### How was it tested?
Printed in ASA and mounted on my Voron 2.4r2, tested with a Samsung A3, Samsung Galaxy mini and a Samsung A52. Working as intended with the appropriate files and for the phones I have available for testing.
### Any background context you want to provide?

### Screenshots (if appropriate)

### ⚠️⚠️**PLEASE READ AND COMPLETE THE CHECKLIST BELOW**⚠️⚠️

  * [ X] I have read the rules available [here](https://github.com/VoronDesign/VoronUsers/wiki/Mod-Submission-Rules) and
        my mod adheres to these rules.
  * [ X] This mod was created by myself and I agree to publish it under the repository
        [license](https://github.com/VoronDesign/VoronUsers/blob/master/LICENSE.md)

MapleLeafMakers reviewed (changes requested):
The jpg files here are *huge*, totalling almost 100MB just in images.  Can you resize those?

MapleLeafMakers commented on printer_mods/AVieira/Universal_Phone_Mount/README.md:
```diff
@@ -0,0 +1,94 @@
+Check "Size and clearence" for more details and to choose what files to print for your needs.
+
+
+![1](https://github.com/AVieira-1/VoronUsers/blob/main/printer_mods/AVieira/Universal_Phone_Mount/Images/Assembly_Mounted_With_Galaxy_A3.jpg?raw=true)
```
When linking to images within your mod folder, use relative links, so that they do not link outside of the VoronDesign repository once merged.  

This one should be 
`![1](Images/Assembly_Mounted_With_Galaxy_A3.jpg)`

MapleLeafMakers commented on printer_mods/AVieira/Universal_Phone_Mount/README.md:
```diff
@@ -0,0 +1,94 @@
+This mount should work with the majority of smartphones out there, I also included some files to hopefully work with tablets if you wish to use one.
+It consists of two claws on a slider that hold your device with the help of an elastic band. 
+
+Check "Size and clearence" for more details and to choose what files to print for your needs.
```
`clearence` should be `clearance`

MapleLeafMakers commented on printer_mods/AVieira/Universal_Phone_Mount/README.md:
```diff
@@ -0,0 +1,94 @@
+
+![1](https://github.com/AVieira-1/VoronUsers/blob/main/printer_mods/AVieira/Universal_Phone_Mount/Images/Slider_Inserts.jpg?raw=true)
+
+The elastic I used is around 60mm in diameter, I looped it twice on the parts. Any elastic should do as long as it's not too thick, you can cut and knot to the desired lenght/strenght.
```
`strenght` should be `strength`

MapleLeafMakers commented on printer_mods/AVieira/Universal_Phone_Mount/LICENSE:
LICENSE files shouldn't be included in your submission.  All mods are required to be licensed under the repository license (GPL3)

AVieira-1 commented:
> The jpg files here are _huge_, totalling almost 100MB just in images. Can you resize those?

Compressed the JPGs with hopefully minimal quality loss. Let me know if it's enough, should total around 5-6mb now.

Corrected the misspellings, sorry about those.

Removed the license file.

Fixed the image links, I had originally /Images/xx.jpg on my first pull but it wasn't working on this one so it got me confused. Works fine without the / at the start.

Thanks

MapleLeafMakers commented:
Clever mod, and looks good to me!

MapleLeafMakers commented:
On closer inspection, I think I agree with the warnings detected by the STL checker, the slider parts would probably print better upside-down.   though they both look perfectly printable to me.

AVieira-1 commented:
> On closer inspection, I think I agree with the warnings detected by the STL checker, the slider parts would probably print better upside-down. though they both look perfectly printable to me.

I oriented the sliders this way because there's an unchamfered lip on the elastic retainer meant to prevent the elastic band from sliding off before installing the claw. Would need supports to be printed if I flipped the part.

I agree that it's technically easier to bridge the smaller wholes instead of the retainer section but I'm not sure the AI is accounting for the unchamfered lip and support requirement.

Let me know if the changes are required, I can easily flip the parts and either chamfer the lip or place some supports.

MapleLeafMakers commented:
No need to change it, I think that's fine.  the bot isn't always right, and they should print perfectly as-is.
